.github/workflows/build_docker_with_args.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
       - name: Build docker image
         run: |
-          ./scripts/build-docker.sh ${{ github.event.inputs.docker_tag }} ${{ github.event.inputs.args }}
+          ./scripts/build-docker.sh ${{ github.event.inputs.docker_tag }} "${{ github.event.inputs.args }}"
```

Comment:
because there are multi args now?

Reply:
> because there are multi args now?

It wouldn't work if the arg contains space, e.g. `--features try-runtime`, you had to write `--features=try-runtime`.
Now with the quotes both would work.